spring-cloud-stream-schema-server/pom.xml

```diff
@@ -27,6 +27,7 @@
 			<groupId>com.h2database</groupId>
 			<artifactId>h2</artifactId>
 			<version>1.4.192</version>
+			<scope>test</scope>
```

Comment:
Is this modification intentional? - i.e. do we want to require them to supply a database? I suppose that's okay.